.github/workflows/cypress-workflow-vanilla-snapshot-based.yml

```diff
@@ -70,8 +70,8 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           working-directory: monetery-test
-          command: yarn run cypress run --browser chrome --headless --spec 'cypress/integration/vanilla-opensearch-dashboards/*.js'
-          wait-on: 'http://localhost:5601'
+          command: yarn run cypress run --browser chrome --headless --env SECURITY_ENABLED=true --spec 'cypress/integration/core-opensearch-dashboards/vanilla-opensearch-dashboards/*.js'
```

Comment:
Line 34 indicates that this is a min artifact. Shall this be `SECURITY_ENABLED=false`?

Reply:
Yes good catch, I'll explicitly change to false.